src/operators/rasterization.md

```diff
@@ -0,0 +1,112 @@
+# Rasterization
+
+The `Rasterization` operator creates a raster from a point vector source.
+It offers two options for rasterization: A grid rasterization and a (gaussian) density rasterization (heatmap).
+
+## Inputs
+
+The `Rasterization` operator expects exactly one _vector_ input.
+
+| Parameter | Type                 |
+|-----------|----------------------|
+| `source`  | `SingleVectorSource` |
+
+## Parameters
+### Grid Rasterization
+
+| Parameter           | Type                | Description                                                | Example Value            |
+|---------------------|---------------------|------------------------------------------------------------|--------------------------|
+| `spatialResolution` | `SpatialResolution` | The spatial resolution of the grid/size of the grid cells. | `{"x": 10.0, "y": 10.0}` |
+| `originCoordinate`  | `Coordinate2D`      | The origin coordinate to which the grid is aligned.        | `{"x": 0.0, "y": 0.0}`   |
+| `gridSizeMode`      | `String`            | The mode how the grid resolution is interpreted.           | `"fixed"`                |
+
+#### Types
+
+The following describes the types used in the grid rasterization parameters.
+
+The parameters `spatialResolution` and `originCoordinate` consist of two fields `x` and `y` which describe a resolution/position in x/y direction.
+
+For `gridSizeMode` the two options `fixed` and `relative` are available.
+`Fixed` means the `spatialResolution` is interpreted as a constant grid cell size. 
+`Relative` means the `spatialResolution` is used as a multiplier for a query's spatial resolution, making the resulting grid size adaptive to the query resolution.
+
+### Density Rasterization
```

Comment:
Maybe say something about the cutoff point

Reply:
https://github.com/geo-engine/docs/pull/35/commits/4fdd1f9cbb6408fbcb8777da60f5059bd12fc97f